### PR TITLE
added sunkit-image recipe

### PR DESCRIPTION
**Pull request by nabobalis**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/sunkit-image/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  license_file: LICENSE.rst
+  summary: A image processing toolbox for Solar Physics
+  description: |
+    sunkit-image is a community-developed, free and open-source solar data image analysis environment for Python
+  doc_url: https://docs.sunpy.org/projects/sunkit-image/en/stable/
+  dev_url: https://github.com/sunpy/sunkit-image
+
+extra:
+  recipe-maintainers:
+    - cadair
```
Comment (Cadair) [inline recipes/sunkit-image/meta.yaml:57]: :+1:

File: recipes/sunkit-image/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+about:
+  home: https://github.com/sunpy/sunkit-image
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.rst
+  summary: A image processing toolbox for Solar Physics
+  description: |
+    sunkit-image is a community-developed, free and open-source solar data image analysis environment for Python
+  doc_url: https://docs.sunpy.org/projects/sunkit-image/en/stable/
```
Comment (Cadair) [inline recipes/sunkit-image/meta.yaml:52]:
```suggestion
  doc_url: https://docs.sunpy.org/projects/sunkit-image/
```

**Comment by nabobalis** [comment]
Hi @conda-forge/help-python, this PR is ready to review.

**Review by carterbox — CHANGES_REQUESTED** [review]
Please check the Windows build. Your tests are failing not because of missing dependencies, but because of missing files during the tests.

I suggest checking the build output from pip to ensure that all package data is being included in the manifest and then checking that sunkit-image doesn't have any hardcoded paths. If you are using hardcoded paths to load resources, then I suggest looking at https://importlib-resources.readthedocs.io/en/latest/ instead of trying to manually guess the location of files.

**Comment by nabobalis** [comment]
Hi @carterbox, sorry for the late reply.

I patched the released version and pushed an update to the recipe. Hopefully its ok now!

**Comment by carterbox** [comment]
Looks like you still have some platform dependent differences in behavior of your package during the tests, but since it looks like you are a developer of sunkit-image, I guess you're OK with that.

**Comment by nabobalis** [comment]
> Looks like you still have some platform dependent differences in behavior of your package during the tests, but since it looks like you are a developer of sunkit-image, I guess you're OK with that.

For now, it is something I plan to address in a refactor down the road.

**Pull request merged**